frontend/components/ClubEditPage/TicketsModal.tsx

```diff
@@ -176,10 +245,17 @@ const TicketsModal = (props: { event: ClubEvent }): ReactElement => {
       const quantities = tickets
         .filter((ticket) => ticket.count != null)
         .map((ticket) => {
+          const usingGroupPricing = ticket.groupDiscount && ticket.groupNumber
           return {
             type: ticket.name,
             count: parseInt(ticket.count || ''),
```

Comment:
`parseInt` and `parseFloat` on empty string returns `NaN`, so maybe default to 0 instead? Minor note: it's better to use nullish comparison(`??`) in place of `||` since `''` will also cause the fallback to be triggered.